rosetta/tests/test_text.py

```diff
@@ -375,12 +375,12 @@ def test_predict_5(self):
         benchmark = pd.Series({'topic_0': 0.5, 'topic_1': 0.5})
         assert_allclose(results.values, benchmark.values, atol=1e-3)
 
-#     def test_predict_6(self):
-#         # Use fact that w0  <--> topic_0,  w1 <--> topic_1
-#         lda = self.choose_lda('lda_2')
-#         tokenized_text = ['newtoken', 'newtoken']
-#         with self.assertRaises(TokenError) as cm:
-#             results = lda.predict(tokenized_text, raise_on_unknown=True)
+    def test_predict_6(self):
```

Comment:
At some point these tests should be renamed to something meaningful...or doc added.  In this case, I don't know why something was supposed to be raised.